hkmc2/shared/src/test/mlscript-compile/quotes/CSPBar.mls

```diff
@@ -0,0 +1,4 @@
+import "../Example.mls"
+module CSPBar with ...
+fun res =
+  ((Example).inc)(0)
```

Comment:
You should be careful when pretty-printing application terms: in JS `x.f(a)` is _not_ the same as `(x.f)(a)`.